src/Model.js

```diff
@@ -9,9 +15,10 @@ export default class Model {
     }
 
     this.configure && this.configure()
-    this[exec_middlewares]('before', 'create')
-    // TODO: create model
-    this[exec_middlewares]('after', 'create')
+    this[execute_middleware]('before', 'create')
+    this[collection] = this.constructor.collection
+    this.id = this[collection].id()
```

Comment:
Did I get it wrong or this id is model-related? If so, ActiveRecord::Base.connection objects do nothing about setting ids for the "models"; ActiveRecord just eases the way the models handle with their own tables' PKs. I'm afraid things get complicated if we focus on mongo right away :worried: